src/gui/mainwindow.h

```diff
@@ -72,6 +77,8 @@ class MainWindow : public QMainWindow {
    */
   MainWindow(QWidget *parent = 0);  // NOLINT [runtime/explicit]
   ~MainWindow();
+  PostureEstimating::PostureEstimator posture;
+  PostureEstimating::PostureEstimator returnEstimator();
```

Comment:
Are these actually used anywhere?

Reply:
Ah this was from an old implementation, i have removed them.